packages/docprovider/src/tokens.ts

```diff
@@ -12,16 +12,6 @@ export const IDocumentProviderFactory = new Token<IDocumentProviderFactory>(
  * An interface for a document provider.
  */
 export interface IDocumentProvider {
-  /**
-   * Resolves to true if the initial content has been initialized on the server. false otherwise.
-   */
-  requestInitialContent(): Promise<boolean>;
```

Comment:
Mind adding a note to the extension migration about these methods being gone from the interface?

https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-3-x-to-4-x